extension/fluentbitextension/README.md

```diff
@@ -0,0 +1,56 @@
+# FluentBit Extension
+
+The fluentbit extension facilitates running a FluentBit subprocess of the
+collector.  This is meant to be used in conjunction with the `fluentforward`
+receiver such that the FluentBit subprocess will be configured to send to the
+TCP socket opened by the `fluentforward` receiver.
+
+You are responsible for providing a configuration to FluentBit via the `config`
+config option.  This will be provided to the subprocess, along with a few other
+config options to enhance the integration with the collector.
+
+
+## Example Config
+
+```yaml
+extensions:
+  health_check:
+  fluentbit:
```

Comment:
The lifecycle of components in Collector is such that extensions are started first, receivers are started last. Can this result in problems if Fluentbit is started but cannot connect to the receiver immediately?

Reply:
It could cause the first batch output from fluentbit to fail, but it would be retried once by default.  I have never actually seen the first batch fail because the flush interval is one second which gives the receiver plenty of time to start up.